File: app/src/main/java/com/grace/placessearch/common/PlacesSearchEnvironmentEnum.java
package com.grace.placessearch.common;

public enum PlacesSearchEnvironmentEnum {

    PROD("https://api.foursquare.com/v2/"),
    STAGING("https://api.foursquare.com/v2/"); // TODO: place holder, please update with env details.

    public String placesBaseUrl;

    PlacesSearchEnvironmentEnum(String placesBaseUrl) {
        this.placesBaseUrl = placesBaseUrl;
    }
}

File: app/src/main/kotlin/com/grace/placessearch/common/PlacesSearchEnvironmentEnum.kt
package com.grace.placessearch.common

enum class PlacesSearchEnvironmentEnum private constructor(// TODO: place holder, please update with env details.

        var placesBaseUrl: String) {

    PROD("https://api.foursquare.com/v2/"),
    STAGING("https://api.foursquare.com/v2/")
}
